openstudiocore/src/shared_gui_components/VariableList.cpp

```diff
@@ -380,7 +383,14 @@ void VariableListController::addItemForDroppedMeasureImpl(QDropEvent * event, bo
       // measure
       analysis::RubyMeasure measure(projectMeasure);
       try{
-        measure.setArguments(m_app->measureManager().getArguments(*project, projectMeasure));
+        auto model = m_app->currentModel();
+        boost::optional<Workspace> workspace;
+        if( model ) {
+          ProgressBar progress(m_app->mainWidget()); 
+          energyplus::ForwardTranslator translator;
+          workspace = translator.translateModel(model.get(),&progress);

```

Comment:
@kbenne maybe these are too picky but since you are already on the scripts tab and the model can't change on the scripts tab, if you cached the workspace in the app you would not have to re translate it here

Also, you shouldn't really have to do this for a model measure but I guess it doesn't hurt